include/myjansson.h

```diff
@@ -7,7 +7,7 @@
 #  include <vector> 
 #  include <cstdlib>
 #  ifndef LEGACY_JANSSON
-#    include "json.hpp"
+#    include "nlohmann/json.hpp"
```

Comment:
Let's go ahead and use angle brackets here for headers not in this repo:
```suggestion
#    include <nlohmann/json.hpp>
```

Reply:
done